indigo_app/views/tasks.py

```diff
@@ -323,3 +325,33 @@ def get_redirect_url(self):
         if self.request.GET.get('next'):
             return self.request.GET.get('next')
         return reverse('task_detail', kwargs={'place': self.kwargs['place'], 'pk': self.kwargs['pk']})
+
+
+class TaskChangeWorkflowsView(TaskViewBase, View, SingleObjectMixin):
+    # permissions
+    permission_required = ('indigo_api.change_task',)
+
+    http_method_names = [u'post']
+    model = Task
+
+    def post(self, request, *args, **kwargs):
+        task = self.get_object()
+        user = self.request.user
+        task.updated_by_user = user
+        ids = self.request.POST.getlist('workflows')
+
+        if ids:
+            workflows = Workflow.objects.filter(country=task.country, locality=task.locality, id__in=ids).all()
+        else:
+            workflows = []
+
+        task.workflows.set(workflows)
+
+        # TODO: timeline?
```

Comment:
@goose-life I should be doing "added to" and "removed from" here, right?

Reply:
https://github.com/laws-africa/indigo/blob/9750dd3b4508e38c8ec1920821db8490d2711411/indigo_app/views/tasks.py#L177-L193

Reply:
@longhotsummer I'm making a PR onto your branch for these